src/app/app.module.ts

```diff
@@ -94,8 +94,7 @@ import {HttpClientModule} from '@angular/common/http'
       provide: NG_ENTITY_SERVICE_CONFIG,
       useValue: {baseUrl: 'https://jsonplaceholder.typicode.com'},
     },
-    DatePipe,
```

Comment:
DatePipe is not needed if you only use it as a pipe in templates. If used as a DI service (like CurrencyPipe), then it should be added to providers.

Reply:
Noted